src/main/resources/twilio-node/imports.mustache

```diff
@@ -7,3 +7,13 @@ import { {{key}} } from "{{apiVersionPath}}/../../../{{value}}";
 {{/vendorExtensions.x-import}}
 {{/vars}}
 {{/responseModel}}
+{{#operations}}
+{{#allParams}}
```

Comment:
Not obvious to me, but how were you able to get rid of the duplicates?

Reply:
The example only has it in a single operation, so it only appears in twilio-node. What tooling should we use to remove the extra? Should I add logic to [clean_unused_imports](https://github.com/twilio/twilio-oai-generator/blob/main/scripts/clean_unused_imports.py)?

Reply:
I'd prefer to use some off-the-shelf tooling like prettier, but doesn't look it supports removing duplicate imports. Given that, I think it's fine to add logic to the import cleaning script.

Reply:
Added [function](https://github.com/twilio/twilio-oai-generator/pull/391/files#diff-7e15ccacf5f960109accbd6a4cca462305184eb56f17bd54ff9de6bbb6ef8475R53) for it